docs/pages/index.js

```diff
@@ -106,9 +106,7 @@ export default function HomePage() {
 
     loadDependencies();
   }, []);
-  const { t } = useSelector(state => ({
-    t: state.options.t,
-  }));
+  const t = useSelector(state => state.options.t);
```

Comment:
Note for self, it's what happens when you work on something without internet to verify it's optimal.

> useSelector() uses strict === reference equality checks by default, not shallow equality (see the following section for more details).

https://react-redux.js.org/next/api/hooks#useselector